File: season3/1주차/막대기/정예림.cs
using System;

namespace algorithm
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int cnt = 0;
            int temp = 0;

            int n = int.Parse(Console.ReadLine());
            int[] num = new int[n];

            for (int i = 0; i < n; i++)
            {
                num[i] = int.Parse(Console.ReadLine());
            }

            for (int i = n-1; i >= 0; i--)
            {
                if (num[i]>temp)
                {
                    temp = num[i];
                    cnt++;
                }

                //if (num[n - 1] < num[i])
                //{
                //    cnt++;
                //}
            }

            Console.WriteLine(cnt);       //맨 오른쪽에 있는 자기자신 +1
        }
    }
}

File: season3/2주차/방 번호/정예림.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main()
{
	int cnt = 1;
	int array[10] = { 0 };
	int n;

	cin >> n;

	while (1)
	{
		array[n % 10]++;
		if (n / 10 == 0)	break;
		n /= 10;
	}

	for (int i = 0; i < 10; i++)
	{
		if (i == 9 || i == 6) continue;
		cnt = max(cnt, array[i]);
	}

	cnt = max(cnt, (array[6] + array[9] + 1) / 2);

	cout << cnt;

	return 0;
}